scvi/train/_trainingplans.py

```diff
@@ -702,15 +708,15 @@ def training_step(self, batch, batch_idx, optimizer_idx=0):
             labelled_tensors=labelled_dataset,
         )
         input_kwargs.update(self.loss_kwargs)
-        _, _, scvi_losses = self.forward(full_dataset, loss_kwargs=input_kwargs)
-        loss = scvi_losses.loss
+        _, _, loss_output = self.forward(full_dataset, loss_kwargs=input_kwargs)
+        loss = loss_output.loss
```

Comment:
is this backwards compatible?

Reply:
Yes? I'm just making the variable naming more consistent